src/sparsezoo/model/utils.py

```diff
@@ -276,6 +276,16 @@ def restructure_request_json(
             file_dict_deployment["file_type"] = "deployment"
             request_json.append(file_dict_deployment)
 
+        # add tokenizer_config.json if available
+        tok_config_dict_training_list = fetch_from_request_json(
+            request_json, "display_name", "tokenizer_config.json"
+        )
+        if len(tok_config_dict_training_list) >= 1:
```

Comment:
In what situations is this available/not available? Would be good to just add "tokenizer_config.json" to the loop above, but I guess tokenizer_config.json is different for some reason?

Reply:
I believe `tokenizer_config.json` is an optional file. It should not be used for detecting if a model is an nlp model because it's optional, but if a model is an nlp model then it should be transferred over to the deployment folder.